Mypy giving error when using emojize

When using emojize inside a typed function, I recieve the following error: 
```bash
error: Call to untyped function "emojize" in typed context  [no-untyped-call]
```
Do you know how to handle that?

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,0 +1,5 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+license_files = LICENSE.txt

--- a/setup.py
+++ b/setup.py
@@ -58,7 +58,7 @@ setup(
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Typing :: Typed'
     ],
-    description="Emoji for Python",
+    description='Emoji for Python',
     keywords=['emoji'],
     extras_require={
         'dev': [
@@ -68,10 +68,19 @@ setup(
         ],
     },
     include_package_data=True,
-    license="New BSD",
+    license='New BSD',
     long_description=readme_content,
     packages=['emoji', 'emoji.unicode_codes'],
-    package_data={"emoji": ["py.typed"]},
+    package_data={
+        'emoji': [
+            'py.typed',
+            '*.pyi',
+        ],
+        'emoji.unicode_codes': [
+            'py.typed',
+            '*.pyi',
+        ],
+    },
     url=source,
     version=version,
     zip_safe=True,

